src/components/toaster/toaster.tsx

```diff
@@ -0,0 +1,26 @@
+import * as RadixToast from '@radix-ui/react-toast';
+import { Toast } from './toast/toast';
+import { useToast } from './toaster-context';
+import classNames from 'classnames';
+
+import styles from './toaster.module.scss';
+
+export function Toaster() {
+    const { toastData: toast, onOpenChange, closeToast } = useToast();
```

Comment:
can't this get list of open toasts and just render all of them in a stack?
then `closeToast` would just remove the toast from the list and `showToast` - add to the list

Reply:
We have `closeToast` and `toast` = showToast. Maybe it is a sense to rename it to `showToast`.